composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "netwerk-digitaal-erfgoed/omeka-s-module-linked-data-sets",
+    "description": "This module provides assistance to use your Omeka-S system to describe and publish your data collections as so called LinkedDataSets, following the guidelines described in NDE Requirements for datasets.",
+    "type": "project",
+    "require-dev": {
+        "phpstan/phpstan": "^1.10",
+        "phpunit/phpunit": "^9.0",
+        "friendsofphp/php-cs-fixer": "^3.15",
+        "laminas/laminas-test": "^4.0",
+        "phpmd/phpmd": "^2.13",
+        "squizlabs/php_codesniffer": "^3.7"
+    },
+    "autoload": {
+        "psr-4": {
+            "NDE\\OmekaSModuleLinkedDataSets\\": "src/",
+            "NDE\\OmekaSModuleLinkedDataSets\\tests\\": "tests/"
+        }
+    },
+    "scripts": {
+        "test": [
+            "@phpstan",
+            "@phpmd",
+            "@phpcs",
+            "@phpunit",
+            "@security-tests"
+        ],
+        "phpmd": "./vendor/bin/phpmd --exclude vendor . text cleancode",
+        "phpcs": "./vendor/bin/phpcs . --ignore=vendor",
+        "phpcpd": "../vendor/bin/phpcpd",
+        "phpunit": "./vendor/bin//phpunit tests/",
+        "phpstan": "./vendor/bin//phpstan analyse -l 9 .",
```

Comment:
Consider adding `--xdebug` to your phpstan call. That should improve performance and rid you of the warning that is currently triggered.

See: `https://phpstan.org/user-guide/command-line-usage#--xdebug`